index.html

```diff
@@ -2331,7 +2331,14 @@ <h3>
 The <a>DID method</a> specification MUST specify how a client can update a
 <a>DID document</a> on the <a>DID registry</a>, including all cryptographic
 operations necessary to establish proof of control, <em>or</em> state that
-updates are not possible.
+updates are not possible. 
+        </p>
+
+        <p>
+An update to a DID is any change, after creation, in the data used to produce a DID Document. 
+<a>DID Method</a> implementers are responsible for defining what constitutes an update, 
+and what properties of the DID Document are supported by a given <a>DID Method</a>. 
+For example, an update operation which replaces key material without changing it could be a valid update that doesn't result in changes to the <a>DID Document</a>.
```

Comment:
```suggestion
For example, an update operation which replaces key material without changing it could be a valid update that does not result in changes to the <a>DID document</a>.
```

This line is also not wrapped to less than 80 characters. Please make sure you wrap to below 80 characters to help people that 1) have line wrapping turned off in their editors on purpose, so they can spot long code lines more easily, and 2) are doing reviews so diff-marking works across a variety of tools, where some of them don't make long lines easy to diff.

As a general rule, avoid contractions in spec text -- contractions tend to be easier for non-native readers to miss and are less formal than we should be in a spec.